### ISSUE TITLE
Fuzzy file finder should prefer direct match

### ISSUE THREAD

**Issue by Valve** [atom/atom#7783]
currently it is not.

When searching for "usa_spec", it first suggests 'user_search_spec.rb' and only after the 'usa_spec.rb'.
It should be the other way around, IMO.

<img width="500" alt="fuzzy" src="https://cloud.githubusercontent.com/assets/27387/8565266/b63d6c50-255c-11e5-8b6a-081ba6c2167d.png">

OSX 10.10.4, Atom 1.0.0

**Comment by thomasjo** [comment]
This is a duplicate of https://github.com/atom/fuzzaldrin/issues/18, and addressed in PR https://github.com/atom/fuzzaldrin/issues/15.
Also related to https://github.com/atom/fuzzy-finder/issues/57.

**Comment by walles** [comment]
See also https://github.com/atom/fuzzaldrin/pull/22.
